docs/formats/zng.md

```diff
@@ -420,7 +420,7 @@ messages provide additional semantics on top of the base ZNG format.
 Any such application-defined messages not known by
 a ZNG data receiver shall be ignored.
 
-The body of control message is JSON, ZSON, ZNG, binary, or UTF-8 text.
+The body of a control message is JSON, ZSON, ZNG, binary, or UTF-8 text.
```

Comment:
Side note: I noticed that the section titles consistently speak of "frames" but then "message" is often used interchangeably within the text. I'm not sure if it might be appropriate to standardize on"frame" everywhere. I didn't try to take that step, but figured I'd point it out in case there's an interest in standardizing.